ui/lib/core/addon/components/ttl-picker2.js

```diff
@@ -12,6 +12,7 @@
  * @param label="Time to live (TTL)" {String} - Label is the main label that lives next to the toggle.
  * @param helperTextDisabled="Allow tokens to be used indefinitely" {String} - This helper text is shown under the label when the toggle is switched off
  * @param helperTextEnabled="Disable the use of the token after" {String} - This helper text is shown under the label when the toggle is switched on
+ * @param description="Longer description about this value, what it does, and why it is useful. Shows up in tooltip next to helpertext"
```

Comment:
I'm not seeing a corresponding .md file for storybook, does the generate for storybook need to be rerun?

Reply:
Yes it does! Thanks for the reminder 😄